android/src/com/frostwire/android/gui/fragments/AboutFragment.java

```diff
@@ -108,10 +113,44 @@ public void onClick(View v) {
             }
         });
 
-        return view;
-    }
+        githubButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                UIUtils.openURL(v.getContext(), Constants.SOCIAL_URL_GITHUB_PAGE + referrerParam);
+            }
+        });
+
+        //Remaining elements including text content
+        final TextView stickersShop = (TextView) view.findViewById(R.id.fragment_about_stickers);
+        final TextView sendFeedback = (TextView) view.findViewById(R.id.fragment_about_feedback);
+        final TextView translateHelp = (TextView) view.findViewById(R.id.fragment_about_translate);
+
+        stickersShop.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                UIUtils.openURL(v.getContext(), Constants.STICKERS_SHOP_URL);
+            }
+        });
+
+        sendFeedback.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                UIUtils.openURL(v.getContext(), Constants.CONTACT_US_URL);
+            }
+        });
+
+        translateHelp.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                UIUtils.openURL(v.getContext(), Constants.TRANSLATE_HELP_URL);
+            }
+        });
```

Comment:
#RefactorProposal
There's so much code repetition here that we could have a method like this:

```java
private initClickUrl(View view, int id, final String clickUrl) {
    View component = view.findById(id);
    if (component != null) {
        component.setOnClickListener(new View.OnClickListener() {
            @Override
             public void onClick(View v) {
                 UIUtils.openURL(v.getContext(), clickURL);
             }
        });
    } 
}
```

Then on your code you just do:
```java
initClickUrl(R.id.fragment_about_facebook_button, Constants.SOCIAL_URL_FACEBOOK_PAGE + referrerParam)
;
initClickUrl(R.id.fragment_about_twitter_button, Constants.SOCIAL_URL_TWITTER_PAGE + referrerParam);
...
initClickUrl(R.id.fragment_about_stickers, Constants.STICKERS_SHOP_URL);
initClickUrl(R.id. fragment_about_feedback, Constants.STICKERS_SHOP_URL);
initClickUrl(R.id. fragment_about_translate, Constants.Constants.TRANSLATE_HELP_URL);
```